### ISSUE TITLE
question about fsod

### ISSUE THREAD

**Issue by 00why00** [Tung-I/Dual-awareness-Attention-for-Few-shot-Object-Detection#5]
When I train the FSOD in your code, the loss is nan

**Comment by 00why00** [comment]
> Thanks for you attention to our work. The training config including learning rate and hyperparameters can be found in the context of the paper. I assume the issue you encountered might be relevant to the learning rate you selected. Please follow the config to see whether the issue can be addressed.

I find this in your paper:

> For thosemodels based on Faster R-CNN [40], we adopt SGD with aninitial learning rate of 0.001, which decays into 0.0001 after 12epochs

So I used the [res50.yml](https://github.com/Tung-I/Dual-awareness-Attention-for-Few-shot-Object-Detection/blob/main/cfgs/res50.yml) as the config of FSOD training which learning rate is 0.001，but the loss is nan
so can you tell me is there any difference between the config of FSOD and DAnA
thank you
